### PR TITLE
Fix Missing Links in Doc

### PR DESCRIPTION
**Pull request by DavidMansolino**
Follow-up #341

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: docs/guide/interfacing-webots-to-third-party-software-with-tcp-ip.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To allow the "tcpip" process to open a different port depending on the controlle
 
 The second advantage is that you can also control a real robot from your third party software by simply implementing your library based on the given remote control library.
 Switching to the remote control mode will redirect the input/output to the real robot through the Inter-Process Communication (IPC).
-An example of remote control is implemented for the e-puck robot in the file "WEBOTS\_HOME/projects/robots/e-puck/worlds/e-puck.wbt" directory of Webots.
+An example of remote control is implemented for the e-puck robot in the file "[WEBOTS\_HOME/projects/robots/e-puck/worlds/e-puck.wbt](https://github.com/omichel/webots/tree/master/projects/robots/e-puck/worlds/e-puck.wbt)" directory of Webots.
```
Comment (fabienrohrer) [inline docs/guide/interfacing-webots-to-third-party-software-with-tcp-ip.md]:
```suggestion
An example of remote control is implemented for the e-puck robot in the file "[WEBOTS\_HOME/projects/robots/gctronic/e-puck/worlds/e-puck.wbt](https://github.com/omichel/webots/tree/master/projects/robots/gctronic/e-puck/worlds/e-puck.wbt)" directory of Webots.
```

File: docs/guide/interfacing-webots-to-third-party-software-with-tcp-ip.md [hunk 2]
```diff
@@ -36,4 +36,4 @@ In the latter case, you may want to run the simulation in real time mode so that
 
 The main drawback of TCP/IP interfacing is that if your robot has a camera device, the protocol must send the images to the controller via TCP/IP, which might be network intensive.
 Hence it is recommended to have a high speed network, or use small resolution camera images, or compress the image data before sending it to the controller.
-This overhead is negligible if you use a low resolution camera such as the Khepera K213 (see example "WEBOTS\_HOME/projects/robots/khepera/khepera1/worlds/khepera\_k213.wbt").
+This overhead is negligible if you use a low resolution camera such as the Khepera K213 (see example "[WEBOTS\_HOME/projects/robots/khepera/khepera1/worlds/khepera\_k213.wbt](https://github.com/omichel/webots/tree/master//projects/robots/khepera/khepera1/worlds/khepera_k213.wbt)").
```
Comment (fabienrohrer) [inline docs/guide/interfacing-webots-to-third-party-software-with-tcp-ip.md]:
```suggestion
This overhead is negligible if you use a low resolution camera such as the Khepera K213 (see example "[WEBOTS\_HOME/projects/robots/k-team/khepera1/worlds/khepera\_k213.wbt](https://github.com/omichel/webots/tree/master/projects/robots/k-team/khepera1/worlds/khepera_k213.wbt)").
```

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
I would add a couple more links (for files like "tcpip.c").

**Comment by DavidMansolino** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:+1:   (Thank you I missed these ones).

**Pull request merged**